cognite/neat/_constants.py

```diff
@@ -200,6 +200,23 @@ def get_asset_read_only_properties_with_connection() -> "list[DMSProperty]":
     return [DMSProperty.model_validate(item) for item in (_ASSET_ROOT_PROPERTY, _ASSET_PATH_PROPERTY)]
 
 
+def get_base_concepts(
+    base_model: Literal["CogniteCore"] = "CogniteCore",
+    no_concepts: int | None = None,
```

Comment:
I read this as **No** concepts. What about?
```suggestion
    concept_count: int | None = None,
```

Reply:
renamed to total_concepts